joomla/joomla-cms
# Fix showon notice

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/13487.

### Summary of Changes

Fix showon notice

### Testing Instructions

1. Apply patch
2. go to Content > Articles > Options and no notice error.

### Documentation Changes Required

None.
